### ISSUE TITLE
Info combining

### ISSUE THREAD

**Issue by DonCicio** [iptv-org/epg#708]
I'm a noob at this. All separate country epg links are working great but is there a way to combine all of them in 1 link?
Thanks

**Comment by freearhey** [comment]
The fact is that different sources have too different information about the same program and it is not at all obvious which version should be given priority. 

Also, sometimes they even indicate different start and end times, which makes this task even more difficult.

These are the main reasons why merging has not yet been done.

**Comment by freearhey** [comment]
For those who will be over this issue in the future:

> GitHub limits the size of files you can track in regular Git repositories.

https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#about-size-limits-on-github

> The deploy step encountered an error: remote: warning: See http://git.io/iEPt8g for more information.        
remote: warning: File guides/us-local/tvtv.us.epg.xml is 55.95 MB; this is larger than GitHub's recommended maximum file size of 50.00 MB 

https://github.com/iptv-org/epg/actions/runs/2908910223

**Comment by freearhey** [comment]
_Update:_

After switching to compressed versions (https://github.com/iptv-org/iptv/discussions/8255), the file size problem described above has gone away. The [largest](https://github.com/iptv-org/epg/blob/gh-pages/guides/us-local/tvtv.us.epg.xml.gz) file is now only 1.74 MB in size.

Unfortunately, the biggest problem (conflicting data from different sources) is still unresolved. But just in case someone decides to work on this issue, I decided to add a JSON version to all the guides to make loading and handling the data a little easier. To load it you just need to change the file extension from `.xml.gz` to `.json`, as shown below:

```
https://iptv-org.github.io/epg/guides/cu/tvcubana.icrt.cu.epg.json
```

I hope this helps!

**Comment by freearhey** [comment]
Implemented in #1212

**Comment by freearhey** [comment]
JSON versions of the guides were removed from the repository in #1979 because some of them started to exceed GitHub's file size [limits](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#file-size-limits).
